MapboxNavigationUI/MGLMapView+MGLNavigationAdditions.h

```diff
@@ -0,0 +1,7 @@
+#import <Mapbox/Mapbox.h>
+
+@interface MGLMapView (MGLNavigationAdditions)
```

Comment:
Also add a `FIXME:` comment noting that this class extension will be reworked following mapbox/mapbox-gl-native#6867.